scaleway/resource_iot_device.go

```diff
@@ -141,18 +130,31 @@ func resourceScalewayIotDevice() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"crt": {
 							Type:        schema.TypeString,
```

Comment:
```suggestion
							Type:        schema.TypeString,
							Required:    true,
```
This should be required

Reply:
As discussed, do not provide a custom certificate is a legit use case, so it is not mandatory.